awscli/topics/config-vars.rst

```diff
@@ -0,0 +1,209 @@
+:title: AWS CLI Configuration Variables
+:description: Configuration Variables for the AWS CLI
+:category: General
+:related command: configure, configure get, configure set
+
```

Comment:
Ok I'll update.  Out of curiosity, do you think these should be pluralized?  It does seem somewhat odd to call these "related command", "related topic", but have them actually accept a csv list.